content/webapp/components/CaptionedImage/CaptionedImage.tsx

```diff
@@ -86,14 +88,17 @@ const CaptionedImage: FunctionComponent<CaptionedImageProps> = ({
         aspectRatio={image.width / image.height}
         hasRoundedCorners={hasRoundedCorners}
       >
+        {isZoomable && <ZoomedPrismicImage image={image} />}
         <ImageWithTasl
           Image={<HeightRestrictedPrismicImage image={image} quality="high" />}
           tasl={{
             ...image.tasl,
             idSuffix: dasherizeShorten(image.contentUrl),
           }}
         />
-        <Caption caption={caption} preCaptionNode={preCaptionNode} />
+        {caption.length > 0 && (
```

Comment:
Does it always have just either 0 or 1 string?

Reply:
it's an array of `prismic.RichText` blocks – I think you recently established that it is currently limited to 1 (although iirc content team might want to allow multiple paras?)

Reply:
What consists of a block? Is it if it was a repeatable zone it'd have more than 1 but because it's no-repeat (within itself) then it's 1 max?

Reply:
no I think it's straightforward (there's a [flag in the text model](https://github.com/wellcomecollection/wellcomecollection.org/blob/main/prismic-model/src/parts/text.ts#L20))